src/main/java/ezschedule/logic/commands/AddCommand.java

```diff
@@ -45,6 +46,8 @@ public CommandResult execute(Model model) throws CommandException {
 
         if (model.hasEvent(toAdd)) {
             throw new CommandException(MESSAGE_DUPLICATE_EVENT);
+        } else if (model.hasEventAtTime(toAdd)) {
+            throw new CommandException(MESSAGE_EVENT_EXIST_AT_TIME);
```

Comment:
Can you add some test case for this situation?

Reply:
Yes, but possible in v1.4 ?